tests/integration/cli/cp_test.py

```diff
@@ -89,3 +89,11 @@ def test_cp_dir_between_containers(self):
         self.run_command('cp busyboxa:/output busyboxb:/output')
         self.assertFileContentsInContainer('busyboxb', '/output/file1.txt', '1')
         self.assertFileContentsInContainer('busyboxb', '/output/file2.txt', '2')
+
+    def test_no_local_file(self):
+        output = self.run_command('cp {} busyboxa:/new-folder/file1.txt'.format('ghost-file'), raise_on_error=False)
+        self.assertInSameLine(output, 'ghost-file', 'does not exist')
```

Comment:
I guess it was already erroring before, we are just making the message nicer? If so I guess this makes sense